tests/test_methods.py

```diff
@@ -74,4 +75,30 @@ def test_reset():
     # reset the obseravtion environment
     new_ob_space, new_info = env.reset()
 
-    assert np.array_equal(new_ob_space, np.array([-1, -1]))
\ No newline at end of file
+    assert np.array_equal(new_ob_space, np.array([-1, -1]))
+
+# test reward function for both greenCrab and greenCrabSimplified
+def test_reward_func():
+    env = greenCrabSimplifiedEnv()
+    env.reset()
+    # self.state = self.self.init_state()
+
+    # test no trap laid for one timestep when no crab
+    action = np.array([-1, -1, -1])
+    assert env.reward_func(action) > 0 # are we expecting positive when no traps laid when no crabs
```

Comment:
Yeah, even if action is [-1, -1,-1], we won't get positive reward strictly for current environment. 
But shouldn't we always encourage positive reward for this action since this will be optimal when there is no crab popluation?

Reply:
Ah I see what you mean! I think discontinuities in the reward function can have some pretty funky effects—this would create a finite reward difference between the action being exactly zero, and the action being infinitesimally small. We could definitely try that out to see what comes out, but because it is just one possible behavior of the reward function I wouldn't 'set it in stone' in a test!

Reply:
Got it. I will change it to >= 0.

Reply:
Nice! Lemme know when you push the changes to approve the PR, all the rest looked pretty solid!